docs/premium-features/break-the-monolith.md

```diff
@@ -100,6 +119,18 @@ This approach allows you to import your existing database structure into Amplica
 To get started with your Prisma schema upload, visit our [Import Prisma Schema](/how-to/import-prisma-schema) documentation for step-by-step instructions.
 -->
 
+## Architecture Change Confirmation
+
+Once you're satisfied with the changes, use the **Apply Plan** button to update your project's architecture.
```

Comment:
I think the screenshot should appear after this line, and maybe we need to elaborate a little regarding the screen displayed